Makefile

```diff
@@ -102,14 +105,14 @@ qa-server: gui
 	bin/pserve qa.ini
 
 .PHONY: run
-run: gui
+run: gui-deps
 	@echo
```

Comment:
I'm not sure this message makes sense here any longer?

Reply:
You do still need to point guiproxy at the parcel runner.